### ISSUE TITLE
EXA briefly displays passphrase / authentication screen on application launch as of 0.7.2

### ISSUE THREAD

**Issue by Vikingtons** [element-hq/element-x-android#3783]
### Steps to reproduce

As of EXA version 0.7.2, users can briefly see the 'Confirm your identity' authentication view when launching the app for the first time. The screen can display for up to three seconds (the delay is oddly inconsistent and may be network related?) before eventually continuing into the conversations view. 

![image](https://github.com/user-attachments/assets/b7ad651e-6d99-40ce-9d23-c81ed5d78c0e)

It's a minor issue but the auth screen can be visible for long enough for a user to interact with (even though they don't need to). This will be visible every time the app is closed from the Android task switcher and reopened, though it will not display when switching out and back into the app, keeping it running in the background.

1. *On a previously established matrix account on EXA (i,e, successfully authenticated and decrypted)
2. Download and install EXA 0.7.2 or newer
3. Launch the application immediately after install
4. Observe 'Confirm your identity screen' on display for several seconds
5. Observe that application will eventually proceed to conversations list

#### What did you expect?
Not to see the 'Confirm your identity' view if this step has already been completed

#### What happened instead?
The 'Confirm your identity' view will display on initial app launch, and will be visible on every app launch

### Your phone model

Google Pixel 7 (Panther)

### Operating system version

Android 14

### Application version and app store

Element X Android (Nightly) 0.7.3

### Homeserver

matrix.org

### Will you send logs?

Yes

### Are you willing to provide a PR?

No
```[tasklist]
### Tasks
- [x] Vik to provide screen recording of defect
- [x] Vik to provide logs via EXA Nightly 0.7.3
```

**Comment by Vikingtons** [comment]
I have a video here, I've not been able to capture a particularly severe case of this yet but hopefully this clip can demonstrate the issue 

https://github.com/user-attachments/assets/1f91362c-2c2f-489d-95d0-2f4719fdb9d4

**Comment by Vikingtons** [comment]
Bug report delivered via EXA 0.7.3

**Comment by Vikingtons** [comment]
This seems to be fixed for me on the Nov 9 0.7.4 nightly release

**Comment by bmarty** [comment]
Thanks. This issue has been fixed by https://github.com/element-hq/element-x-android/pull/3830.
